src/seedu/addressbook/AddressBook.java

```diff
@@ -557,6 +581,30 @@ private static String executeListAllPersonsInAddressBook() {
     private static void executeExitProgramRequest() {
         exitProgram();
     }
+    
+    /**
+     * Sorts persons in the address book by their lexicographical order.
+     * 
+     * @return Arraylist sorted in alphabetical order
+     */
+    
+    private static String executeSortAllPersonsInAddressBook() {
+        // TODO add example to help
+    	
+    	ArrayList<String[]> sortPersonsAlphabetically = getAllPersonsInAddressBook();
+   
+    	Collections.sort(sortPersonsAlphabetically, new Comparator<String[]>(){
+    		public int compare(String[] o1, String[] o2){
+    			return o1[PERSON_DATA_INDEX_NAME].compareTo(o2[PERSON_DATA_INDEX_NAME]);
+    			
+    		}
+    	});
```

Comment:
When you modify the behavior, remember to update documentation and tests as well.